test/client/fixtures/runner/automation/select-element-test.js

```diff
@@ -1361,10 +1429,27 @@ $(document).ready(function () {
         select.selectedIndex = 0;
         equal(select.selectedIndex, 0);
 
-        clickPlaybackAutomation($option[0], {}, function () {
-            equal(select.selectedIndex, 2);
-            equal(handlersLog.join(), browserUtils.isIE ? 'select mousedown,select mouseup,select change,select click' : 'option mousedown,option mouseup,select change,option click');
-            startNext();
+        window.async.series({
```

Comment:
As I see the test for `select with size more than one`. Should we ignore this for iOS and Android